node install error on windows 10

$ cd samples/

GE Laptop3@GELaptop3-PC /cygdrive/c/jinwork/samples
$ npm install
npm WARN deprecated lodash@0.9.2: Grunt needs your help! See https://github.com/                             gruntjs/grunt/issues/1403.
npm WARN deprecated graceful-fs@1.2.3: graceful-fs version 3 and before will fai                             l on newer node releases. Please update to graceful-fs@^4.0.0 as soon as possibl                             e.

> travis-multirunner@3.0.0 postinstall C:\jinwork\samples\node_modules\webrtc-ut                             ilities\node_modules\travis-multirunner
> ./bin/travis-sync

'.' is not recognized as an internal or external command,
operable program or batch file.
npm WARN deprecated npmconf@2.1.2: this package has been reintegrated into npm a                             nd is now out of date with respect to npm

> chromedriver@2.21.2 install C:\jinwork\samples\node_modules\webrtc-utilities\n                             ode_modules\chromedriver
> node install.js

Downloading http://chromedriver.storage.googleapis.com/2.21/chromedriver_win32.z                             ip
Saving to C:\cygwin64\tmp\chromedriver\chromedriver_win32.zip
Receiving...
Received 781K...
Received 1563K...
Received 2345K...
Received 2537K total.
Extracting zip contents
Copying to target path C:\jinwork\samples\node_modules\webrtc-utilities\node_mod                             ules\chromedriver\lib\chromedriver
Done. ChromeDriver binary available at C:\jinwork\samples\node_modules\webrtc-ut                             ilities\node_modules\chromedriver\lib\chromedriver\chromedriver.exe
npm ERR! Windows_NT 10.0.10586
npm ERR! argv "C:\Program Files (x86)\nodejs\node.exe" "C:\Program Files (x8                             6)\nodejs\node_modules\npm\bin\npm-cli.js" "install"
npm ERR! node v4.4.1
npm ERR! npm  v2.14.20
npm ERR! code ELIFECYCLE

npm ERR! travis-multirunner@3.0.0 postinstall: `./bin/travis-sync`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the travis-multirunner@3.0.0 postinstall script './bin/travis                             -sync'.
npm ERR! This is most likely a problem with the travis-multirunner package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ./bin/travis-sync
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs travis-multirunner
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!
npm ERR!     npm owner ls travis-multirunner
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\jinwork\samples\npm-debug.log

--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "test": "testling -x ./start.sh",
-    "postinstall": "./bin/travis-sync"
+    "postinstall": "node bin/travis-sync"
   },
   "repository": {
     "type": "git",

